MongoBundle/Resources/config/transformer.yml

```diff
@@ -1,2 +1,7 @@
 parameters:
     open_orchestra.transformer.boolean_to_bdd.class: OpenOrchestra\Transformer\ConditionFromBooleanToMongoTransformer
+services:
+    open_orchestra.transformer.boolean_to_bdd:
+        class: %open_orchestra.transformer.boolean_to_bdd.class%
+        arguments:
+            - ''
```

Comment:
so why no refacto this service to a default value for the argument in the constructor or always use this service with a setter